bokehjs/src/lib/core/util/math.ts

```diff
@@ -75,26 +71,6 @@ export function to_radians_coeff(units: AngleUnits): number {
   }
 }
 
-// http://www2.econ.osaka-u.ac.jp/~tanizaki/class/2013/econome3/13.pdf (Page 432)
-export function rnorm(mu: number, sigma: number): number {
-  // Generate a random normal with a mean of 0 and a sigma of 1
-  let r1: number
-  let r2: number
-  while (true) {
-    r1 = random()
-    r2 = random()
-    r2 = (2*r2-1)*Math.sqrt(2*(1/Math.E))
-    if (-4*r1*r1*Math.log(r1) >= r2*r2)
-      break
-  }
-  let rn = r2/r1
-
-  // Transform the standard normal to meet the characteristics that we want (mu, sigma)
-  rn = mu + sigma*rn
-
-  return rn
-}
```

Comment:
@ianthomas23, can you help me out with understanding if I can replace this function with `AbstractRandom.normals()` (see below)? In general, can you review changes related to random number generation.

Reply:
Yes, you can replace this with `AbstractRandom.normals()`, the two functions are equivalent. This looks unnecessarily complicated whereas the Box-Muller method in `AbstractRandom.normals()` can be explained using simple geometry in polar coordinates and has a wikipedia page (https://en.wikipedia.org/wiki/Box%E2%80%93Muller_transform).

Reply:
Good, this way we will have one source of random number generation in `core/util/random`.